registration-processor/pre-processor/registration-processor-packet-validator-stage/src/main/java/io/mosip/registration/processor/stages/validator/impl/PacketValidatorImpl.java

```diff
@@ -162,7 +176,29 @@ public boolean validate(String id, String process, PacketValidationDto packetVal
         return packetValidationDto.isValid();
     }
 
-    private boolean individualBiometricsValidation(String id, String process) throws RegistrationProcessorCheckedException {
+    private boolean xsdValidation(String id, String process, PacketValidationDto packetValidationDto) throws ApisResourceAccessException, PacketManagerException, JsonProcessingException, IOException, RegistrationProcessorCheckedException {
```

Comment:
call this method as biometricsXSDValidation

Reply:
I have changed it.